### PR TITLE
fix(@community): Fix community join bubble

### LINKED ISSUE #6133: community invite disappears after restart
# Bug Report

## Steps to reproduce

1. send or receive a community invite
2. restart app
3. go back to the 1 on 1 chat

result: invite is now blank

Screen Shot 2022-06-17 at 2 17 24 PM

### PR DESCRIPTION
**Pull request by alaibe**
fixes #6133

When restarting the app, the community is not in the list
and data needs to be requested

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]

File: ui/imports/shared/views/chat/InvitationBubbleView.qml [hunk 1]
```diff
@@ -42,14 +44,23 @@ Item {
...
+        onCommunityChanged: function (communityId) {
+            if (communityId === root.communityId) {
+                root.invitedCommunity = getCommunity()
+            }
+        }
+    }
+
+    Connections {
+        target: root.store.communitiesModuleInst
+        onCommunityAdded: function (communityId) {
```
Comment (saledjenic) [inline ui/imports/shared/views/chat/InvitationBubbleView.qml:58]: @alaibe in these cases you could just add `onCommunityAdded` slot to the previous connections object, that's possible as long as `target` refers to the object that sends the signal you want to connect to.

**Review by saledjenic — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]

**Comment by alaibe** [comment]
@Hbouaz what you are seeing are problem with the mailserver being flakey, not the code itself

**Comment by Hbouaz** [comment]
Note that the problem is still current:
Just after sending an invite:

image

Then after restarting the app the bubble is lost:

image


Also note:
Sending a community invite from master to PR branch shows the bubble on master only
Sending a community invite from PR to master branch shows the bubble on both (which then disappears on PR branch after the restart, as is the subject of this ticket)

Some of these issues might be related to mailserver like you said @alaibe, but this doesn't allow the validation of the fix

**Comment by Hbouaz** [comment]
Hi @alaibe 
Any updates om my comments above, were you able to reproduce the same?

**Comment by alaibe** [comment]
@Hbouaz all the problem you mentioned, the fact that this is flakey are caused by the mailserver not returning the expected that, I put that on hold til we have a solution for the mailserver

**Comment by Hbouaz** [comment]
@alaibe does this mean the PR will remain open and once those issues are fixed you rebase and we have another go at it? 

In that case will you add a mention or ping me when it is ready

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**